viewing_party/party.py

```diff
@@ -0,0 +1,226 @@
+        unique_movies_friends.append({"title": movie})
+    
+    return unique_movies_friends
+
+def get_available_recs(user_data):
+    movies_watched = []
+    friends_watched = []
+    movies_recommended = []
+
+    for movie in user_data["watched"]:
+        movies_watched.append(movie)
+
+    for friend in user_data["friends"]:
+        for movie in friend["watched"]:
+            friends_watched.append(movie)
+
+    for user_movie in movies_watched:
+        found = []
+        for friend_movie in friends_watched:
+            if friend_movie["title"] == user_movie["title"]:
+                found.append(friend_movie)
+        
+        for found_movie in found:
+            friends_watched.remove(found_movie)
+        # if movie in movies_watched:
+        #     friends_watched.remove(movie)
+    
+    for friend_movie in friends_watched:
+        if friend_movie["host"] in user_data["subscriptions"] and friend_movie not in movies_recommended:
+            movies_recommended.append(friend_movie)
+    
+    return movies_recommended
+
+    
+    
+
+
+    #iterate through what friends have watched
+    # create a list that describes the movies for recommend
+    # if movie title is in watched and host is in subscriptions, then add it to movies_recommended
+    # return a list of dictionaries 
+
+def get_new_rec_by_genre(user_data):
+    movies_watched = []
+    friends_watched = []
+    movies_recommended = []
+
+    for movie in user_data["watched"]:
+        movies_watched.append(movie)
+
+    for friend in user_data["friends"]:
+        for movie in friend["watched"]:
+            friends_watched.append(movie)
+
+    for user_movie in movies_watched:
+        found = []
+        for friend_movie in friends_watched:
+            if friend_movie["title"] == user_movie["title"]:
+                found.append(friend_movie)
+        
+        for found_movie in found:
+            friends_watched.remove(found_movie)
+
+    genre_count = {}
+    for movie in user_data["watched"]:
+        if movie["genre"] in genre_count:
+            genre_count[movie["genre"]] += 1
+        else:
+            genre_count[movie["genre"]] = 1
+    
+    max_value = 0
+    max_genre = None
+
+    for genre in genre_count:
+        if genre_count[genre] > max_value:
+            max_value = genre_count[genre]
+            max_genre = genre
+
+    for friend_movie in friends_watched:
+        if max_genre == friend_movie["genre"] and friend_movie not in movies_recommended:
+            movies_recommended.append(friend_movie)
+
+    return movies_recommended
+
+def get_rec_from_favorites(user_data):
+    unique_favorites = get_unique_watched(user_data)
+    recommended_movies = []
+
+    for movie in user_data["favorites"]:
+        for favorite_movie in unique_favorites:
+            if favorite_movie == movie:
+                recommended_movies.append(movie)
+
+    return recommended_movies
+
+
+    
+    
+    
+    
+    
+    
```

Comment:
```suggestion
```

You don't need all of this whitespace. To make your project neater and follow pep8 guidelines, you can have VS Code autoformat your files by right-clicking on a file in VS Code and select Format Document With... then select python from the drop-down menu. You'll then be prompted to install autopep8, click yes and the package will be installed within your virtual environment.